app/src/main/java/com/amazonaws/kinesisvideo/demoapp/fragment/StreamWebRtcConfigurationFragment.java

```diff
@@ -231,6 +236,19 @@ public void onClick(final View view) {
     }
 
     private void startViewerActivity() {
+        Log.i(TAG, "Start Viewer button clicked - beginning viewer activity setup");
+        final SparseBooleanArray checked = mOptions.getCheckedItemPositions();
+        
+//        Check if Ingest Media is checked with Send Video
+        if (mIngestMedia.isChecked() && checked.get(0)) {
+            new AlertDialog.Builder(getActivity())
+                    .setPositiveButton("OK", null)
```

Comment:
Multi-viewer participant is allowed to send only audio;  please deselect video for multiviewer option